frontend/src/main/webapp/create_trip.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Trip Me Out</title>
+    <script src="create_trip.js"></script>
+  </head>
+  <body>
+    <h1>Trip Me Out!</h1>
+    <h2>Create Trip</h2>
+    <input id="location" placeholder="Location..." type="text"/>
+    <input id="radius" placeholder="Radius" type="text"/>
```

Comment:
maybe make this a number input or drop down menu since we want radius to be between 0 and 50,000 meters (upper limit imposed by places api)

Reply:
A drop-down menu might be onerous to use, but I second the idea of using a [number input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number).

Also, please include a unit for the radius in the description.